### DISCUSSION TITLE
Override variables by using builder

### DISCUSSION THREAD

**Discussion by konraddysput** [callstack/react-native-builder-bob#474] [Q&A] [answered]
Hi! 

I'm creating a new react-native library and I'm using react-native-builder-bob. In my library, I want to use information from package.json - for example, name and version to dynamically include information about the reporter in the output. I can do it via manual override in Babel, however, instead of overriding your Babel configuration I would prefer to extend it just by adding a plugin to replace specific strings in my app during the build process. 

Based on this: 
https://github.com/callstack/react-native-builder-bob/blob/8c0cbae3e149be24f50e1afc5168e5ab634a4f37/packages/react-native-builder-bob/src/utils/compile.ts#L84

I believe it's not possible to do such a thing, right? Do you have any other recommendations on how can I set a variable during the build stage? Do you have a recommendation on how can I extend your babel configuration?

**Comment by satya164** [comment, marked as answer]
Currently, it's not possible to extend the babel configuration. In future, we plan to export the config so consumers can extend it.

**Reply by konraddysput** [reply]
Where can I found your configuration so I can have the same settings you have, but with small changes I want to apply?

**Reply by satya164** [reply]
You linked to it in your post
